scripts/set-backend-env-vars.sh

```diff
@@ -35,19 +35,8 @@ fi
 
 echo "Setting environment variables for $CGAPPNAME_BACKEND"
 
-cf set-env "$CGAPPNAME_BACKEND" ACR_VALUES "$ACR_VALUES"
```

Comment:
These are now optional environment variables. They have non-secret defaults applied in the respective settings modules but if needed can be overridden by environment variables on the Cloud.gov application.